### PR TITLE
(GH-1) Basic implementation

### LINKED ISSUE #1: Basic implementation
Basic implementation:

* Issue provider which can parse a file for markdownlint output (expect output with `options.resultVersion set to 1)

### PR DESCRIPTION
**Pull request by pascalberger**
Basic implementation for reading issues from Markdownlint.

Better URL linking possible if https://github.com/DavidAnson/markdownlint/pull/54 gets merged.

Requires #3 to be merged first and then rebased

Fixes #1

### CODE REVIEW

File: src/Cake.Prca.Issues.Markdownlint/MarkdownlintProviderAliases.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            FilePath logFilePath)
+        {
+            context.NotNull(nameof(context));
+            logFilePath.NotNull(nameof(logFilePath));
+
+            return context.Markdownlint(MarkdownlintSettings.FromFilePath(logFilePath));
+        }
+
+        /// <summary>
+        /// Gets an instance of a provider for code analysis issues reported by JMarkdownlint using log file content.
```
Comment (stffabi) [inline src/Cake.Prca.Issues.Markdownlint/MarkdownlintProviderAliases.cs]: Typo: JMarkdownLint

File: src/Cake.Prca.Issues.Markdownlint/MarkdownlintSettings.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        /// <summary>
+        /// Gets the content of the log file.
+        /// </summary>
+        public string LogFileContent { get; private set; }
+
+        /// <summary>
+        /// Returns a new instance of the <see cref="MarkdownlintSettings"/> class from a log file on disk.
+        /// </summary>
+        /// <param name="logFilePath">Path to the Markdownlintlog file.</param>
```
Comment (stffabi) [inline src/Cake.Prca.Issues.Markdownlint/MarkdownlintSettings.cs]: Typo: Markdownlint log

File: src/Cake.Prca.Issues.Markdownlint/MarkdownlintProviderAliases.cs (hunk 1 above)
Comment (pascalberger) [inline src/Cake.Prca.Issues.Markdownlint/MarkdownlintProviderAliases.cs]: Fixed

File: src/Cake.Prca.Issues.Markdownlint/MarkdownlintSettings.cs (hunk 2 above)
Comment (pascalberger) [inline src/Cake.Prca.Issues.Markdownlint/MarkdownlintSettings.cs]: Fixed

**Pull request merged**